content/en/docs/ambient/architecture/hbone/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: HBONE
```

Comment:
Consider renaming to "TLS & Tunneling" and define HBONE within the content here.

When we link to this doc from elsewhere we will just mention TLS